src/components/FormTextArea.vue

```diff
@@ -20,9 +20,11 @@
       :value="value"
       @input="$emit('input', $event.target.value)"
     />
-    <div v-if="(validator && validator.errorCount) || error" class="invalid-feedback">
+    <div v-if="error" class="invalid-feedback">
+      <div >{{error}}</div>
```

Comment:
I don't think this needs to be wrapped in another `div`. I think it will work fine like this, and avoids nesting the extra `div`:
```html
<div v-if="error" class="invalid-feedback">{{error}}</div>
```

Reply:
I changed it by creating a component to show the errors